Doc/whatsnew/3.12.rst

```diff
@@ -210,6 +210,11 @@ array
 asyncio
 -------
 
+* :mod:`asyncio` is optimized to avoid excessive copying when writing to socket
+  and uses :meth:`~socket.socket.sendmsg` if the platform supports it.
+  A speedup of 80% was observed on Linux on simple tcp echo server benchmark.
```

Comment:
Number is taken from `asyncio_tcp` benchmark from https://github.com/faster-cpython/ideas/blob/main/results/bm-20230402-3.12.0a6%2B-385b5d6/bm-20230402-linux-x86_64-python-385b5d6e091da454c3e0-3.12.0a6%2B-385b5d6-vs-3.10.4.md

Reply:
@AlexWaygood I welcome your suggestions to better document it.